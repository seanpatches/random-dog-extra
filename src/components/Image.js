import React from 'react';
import PropTypes from 'prop-types';

function Image({ img }) {
  return <img src={img} />;
}

Image.propTypes = {
  img: PropTypes.string.isRequired
};

export default Image;
