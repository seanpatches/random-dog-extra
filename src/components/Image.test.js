import React from 'react';
import { shallow } from 'enzyme';
import Image from './Image';

describe('Image component', () => {
  it('renders image comp', () => {
    const img = 'https://images.dog.ceo/breeds/shiba/shiba-6.jpg';
    const wrapper = shallow(<Image img={img}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
