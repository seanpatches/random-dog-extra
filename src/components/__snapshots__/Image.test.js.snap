// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component renders image comp 1`] = `
<img
  src="https://images.dog.ceo/breeds/shiba/shiba-6.jpg"
/>
`;
